Code/LauncherUnified/Launcher.cpp

```diff
@@ -570,6 +573,16 @@ namespace O3DELauncher
             status = ReturnCode::ErrCrySystemInterface;
         }
 
+    #if !defined(_RELEASE)
+        // until CrySystem can be removed (or made to be managed by the component application),
+        // we need to manually clear the BudgetTracker before CrySystem is unloaded so the Budget
+        // pointer(s) it has references to are cleared properly
+        if (auto budgetTracker = AZ::Interface<AZ::Debug::BudgetTracker>::Get(); budgetTracker)
+        {
+            budgetTracker->Reset();
+        }
+    #endif // !defined(_RELEASE)
+
```

Comment:
Shouldn't this be in the `#if !defined(AZ_MONOLITHIC_BUILD)` block below?

Reply:
Ah right, good point